app/views/provider_mailer/application_rejected_by_default.text.erb

```diff
@@ -4,7 +4,7 @@ Dear <%= @provider_user_name || 'colleague' %>
 
 <%= @application.candidate_name %> submitted an application for <%= @application.course_name_and_code %> on <%= @application.submitted_at %>.
 
-This application has been rejected automatically because you did not respond within <%= @application.rbd_days %> working days.
```

Comment:
Feature flags? Or we not meant to use feature flags on these?

Reply:
I wasn't sure, but i think you're right actually as things will just return to normal (hopefully) afterwards. Adding them noiw

Reply:
Cool, I wasn't sure If I'm over complicating things 😅